### PR TITLE
Vultr: Introducing vultr_network module 

### PR DESCRIPTION
**Pull request by Spredzy**
##### SUMMARY

This commit introduces a new module called vultr_network.

It allows a user to manage networks on the Vultr cloud.

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

  - vultr_network

##### ANSIBLE VERSION

  - devel

##### ADDITIONAL INFORMATION

  - N/A

### CODE REVIEW

File: lib/ansible/modules/cloud/vultr/vultr_network.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+options:
+  name:
+    description:
+      - Name of the network.
+    required: true
+    aliases: [ description, label ]
+  cidr:
+    description:
+      - The CIDR IPv4 network block to be used when attaching servers to this network
+    required: true
```
Comment (resmo) [inline lib/ansible/modules/cloud/vultr/vultr_network.py]: please remove `required: true`and replace with a description:

~~~
- Required if I(state=present).
~~~

File: lib/ansible/modules/cloud/vultr/vultr_network.py [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+  region:
+    description:
+      - Region the network is deployed into.
+    required: true
```
Comment (resmo) [inline lib/ansible/modules/cloud/vultr/vultr_network.py]: please remove `required: true`and replace with a description:

~~~
- Required if I(state=present).
~~~

**Review by resmo — CHANGES_REQUESTED** [review]
Thanks @Spredzy for your hard work! minor issues in doc, needs rebase. well done!

**Comment by Spredzy** [comment]
@resmo Thanks :) All the comments have been adressed.

**Review by resmo — APPROVED** [review]
shipit

**Pull request merged**